cmd/xl-v1-metadata.go

```diff
@@ -173,7 +176,8 @@ func newXLMetaV1(object string, dataBlocks, parityBlocks int) (xlMeta xlMetaV1)
 // IsValid - tells if the format is sane by validating the version
 // string and format style.
 func (m xlMetaV1) IsValid() bool {
```

Comment:
similar check is needed for fs too. i.e older binaries should fail reading from newer xl.json